packages/@glimmer/compiler/lib/wire-format-debug.ts

```diff
@@ -270,6 +264,19 @@ export default class WireFormatDebugger {
     }
   }
 
+  private formatCurryType(value: number) {
+    switch (value) {
+      case CurriedType.Component:
+        return 'component';
+      case CurriedType.Helper:
+        return 'helper';
+      case CurriedType.Modifier:
+        return 'modifier';
+      default:
+        throw new Error('unknown curry type');
```

Comment:
Can you make this an `exhausted()` type thing (with a `never`, so that we can fail type checking if a new `CurriedType` is added that isn't accounted for)?

Reply:
👍